File: app/src/main/java/jp/java_conf/ikastyle/Entity/HeadGear.java
package jp.java_conf.ikastyle.Entity;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;

@Entity(tableName = "MAST_HEAD_GEAR", primaryKeys = {"id", "language_code"})
public class HeadGear {
    @ColumnInfo(name = "id")
    public int id;

    @ColumnInfo(name = "language_code")
    public int languageCode;

    @ColumnInfo(name = "name")
    @NonNull
    public String name;
}

File: app/src/main/java/com/splatool/ikastyle/model/data/entity/HeadGear.kt
package com.splatool.ikastyle.model.data.entity

import androidx.annotation.NonNull
import androidx.room.*

@Entity(tableName = "MAST_HEAD_GEAR", primaryKeys = ["id", "language_code"])
data class HeadGear (
    @ColumnInfo(name = "id") val id : Int,
    @ColumnInfo(name = "language_code") val languageCode : Int,
    @ColumnInfo(name = "name") @NonNull val name: String
)